sys/shell/shell.c

```diff
@@ -113,8 +113,8 @@ static shell_command_handler_t search_commands_xfa(char *command)
     return NULL;
 }
 
-static shell_command_handler_t find_handler(
-        const shell_command_t *command_list, char *command)
+shell_command_handler_t shell_find_handler(const shell_command_t *command_list,
+                                           const char *command)
 {
     shell_command_handler_t handler = NULL;
 
```

Comment:
(Because this is a public api now, protect against developer errors)
```suggestion
assert(command);
```